lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/MethodProcessor.java

```diff
@@ -159,36 +159,42 @@ private void generateCode(MethodDefinition methodDefinition) throws IOException
       out.println("  description = \"\"\"\n" + methodDefinition.getDescription() + "\"\"\")");
       out.println("public class " + methodDefinition.getClassName() + " extends BuiltinRootNode {");
       out.println("  private @Child " + methodDefinition.getOriginalClassName() + " bodyNode;");
-      out.println("  private final boolean staticOfInstanceMethod;");
+      out.println("  private static final class Internals {");
```

Comment:
Encapsulating all the state into `Internals` class that can be used by `BuiltinRootNode` as well as `InlinedCallNode`.